### ISSUE TITLE
getting error installing on ubuntu 18.04

### ISSUE THREAD

**Issue by xingfanxia** [hyperq/jav#3]
hi, getting this error on my setup
```
package jav/cmd: unrecognized import path "jav/cmd" (import path does not begin with hostname)```

**Comment by xingfanxia** [comment]
And if I execute from the dir inside GOPATH 
still getting an error
```
go: finding github.com/hyperq/jav latest
go get: github.com/hyperq/jav@v0.0.0-20200313080411-3402e5c5549b: parsing go.mod:
	module declares its path as: jav
	        but was required as: github.com/hyperq/jav
```

**Comment by hyperq** [comment]
fixed [8806fe813973d7a3cbf6a33c94023d6c274f69e4](https://github.com/hyperq/jav/commit/8806fe813973d7a3cbf6a33c94023d6c274f69e4)
